packages/fluentui/react-northstar/src/themes/teams/components/SvgIcon/svgIconStyles.ts

```diff
@@ -97,6 +97,10 @@ const svgIconStyles: ComponentSlotStylesPrepared<SvgIconStylesProps, SvgIconVari
       }),
 
       transform: `rotate(${rotate}deg)`,
+      // Use for Emotion:
```

Comment:
So do we need this?

Reply:
No, I will apply a change there, stay tuned ✍️

Reply:
Fixed in `bfb1932`.